frontend/src/components/form/InputMask.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import { toRef } from 'vue';
+import { useField, ErrorMessage } from 'vee-validate';
+
+import { InputMask } from '@/lib/primevue';
+
+// Props
+type Props = {
+  helpText?: string;
+  label?: string;
+  name: string;
+  mask: string;
+  placeholder?: string;
+  disabled?: boolean;
+  bold?: boolean;
+};
+
+const props = withDefaults(defineProps<Props>(), {
+  helpText: '',
+  type: 'text',
+  label: '',
+  placeholder: '',
+  disabled: false,
+  bold: true
+});
+
+const { errorMessage, value } = useField<string>(toRef(props, 'name'));
+</script>
+
+<template>
+  <div class="field col">
+    <label
+      :class="{ 'font-bold': bold }"
+      :for="name"
+    >
+      {{ label }}
+    </label>
+    <InputMask
+      v-model.trim="value"
+      :aria-describedby="`${name}-help`"
+      :name="name"
+      :mask="mask"
+      :placeholder="placeholder"
+      class="w-full"
+      :class="{ 'p-invalid': errorMessage }"
+      :disabled="disabled"
+    />
+    <small :id="`${name}-help`">{{ helpText }}</small>
+    <div>
+      <ErrorMessage :name="name" />
+    </div>
```

Comment:
Is the new div wrapper addressing some kind of strange spacing/layout issue? Usually we try to avoid nesting more DOM elements whenever possible.